bcp47/library/Data/BCP47/Internal/Arbitrary.hs

```diff
@@ -44,7 +44,8 @@ alphaNumChars :: String
 alphaNumChars = alphaChars ++ numChars
 
 alphaChars :: String
-alphaChars = ['a' .. 'z'] ++ ['A' .. 'Z']
+alphaChars = filter (`notElem` "xX") allAlphaChars
```

Comment:
Avoid generating `x` which can trigger the private use parser

Reply:
This would be useful as an inline comment.